test/SlowTests/Server/Documents/SqlMigration/NpgSQLSchemaTest.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Newtonsoft.Json.Linq;
+using Raven.Server.ServerWide.Context;
+using Raven.Server.SqlMigration;
+using Raven.Server.SqlMigration.Model;
+using Raven.Server.SqlMigration.Schema;
+using Xunit;
+
+namespace SlowTests.Server.Documents.SqlMigration
+{
+    public class NpgSQLSchemaTest : SqlAwareTestBase
+    {
+        [Fact]
```

Comment:
How do you only run it if we have postgres available?

Reply:
I think you need `RequiresNpgSqlFactAttribute ` here

Reply:
done